backoffice/src/panels/beaconSummaryPanel/BeaconSummaryPanel.tsx

```diff
@@ -61,22 +62,24 @@ export const BeaconSummaryPanel: FunctionComponent<IBeaconSummaryProps> = ({
     switch (state) {
       case DataPanelStates.Viewing:
         return (
-          <>
+          <OnlyVisibleToUsersWith role={"UPDATE_RECORDS"}>
```

Comment:
Check BeaconController endpoint for update_record

Reply:
This looks to be hiding the entire summary panel behind the role, not just the edit functionality?